domains/certificates/Query.API/API/appsettings.Development.json

```diff
@@ -10,5 +10,11 @@
     },
     "Datasync": {
         "Url": "http://data-sync-api"
+    },
+    "RabbitMq": {
+        "Host": "localhost",
+        "Port": "5672",
+        "Username": "guest",
+        "Password": "guest"
```

Comment:
What about our docker-compose setup? It is not part of this PR, so I guess that these values will only work if you manually start a rabbitmq instance.